eth/backend.go

```diff
@@ -142,6 +143,13 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 	}
 	log.Info("Initialised chain configuration", "config", chainConfig)
 
+	// Start the precomputation of Lagrange points
+	// if this config supports verkle trees.
+	if chainConfig.CancunBlock != nil {
+		log.Info("Detected the use of verkle trees, rebuilding the cache")
+		verkle.GetConfig()
+	}
```

Comment:
This should no longer be necessary, except maybe for the log... but the message should be changed to "rebuilding the cache" so that if it gets stuck, people will see this as the last message